pkg/config/config.go

```diff
@@ -241,6 +241,11 @@ func initConfig(config Config) {
 		if pathExists("/host/proc") {
 			config.SetDefault("procfs_path", "/host/proc")
 			config.SetDefault("container_proc_root", "/host/proc")
+
+			// Used by some librairies (like gopsutil)
+			if v := os.Getenv("HOST_PROC"); v == "" {
+				os.Setenv("HOST_PROC", "/host/proc")
```

Comment:
Should we also do this in the `else` case ?

Reply:
So this sets `HOST_PROC` to `/host/proc` if the env var isn't set, what should the `else` clause do in this case per your comment?

Reply:
true ... :facepalm: